### DISCUSSION TITLE
Ok to write about Copilot publicly?

### DISCUSSION THREAD

**Discussion by mlapeter** [community/community#8042] [Copilot Conversations] [answered]
I'm exploring Copilot by using it as much as possible to create a simple choose your own adventure game from scratch. So far I've been pretty blown away! I checked docs but couldn't find any guidance on blogging publicly about Copilot, and if there are any restrictions beyond the standard code of conduct. The repo for anyone curious is here: https://github.com/mlapeter/ronnieregret and the in-progress demo of it is here: https://www.ronnieregret.com/

**Comment by hsenag** [comment, marked as answer]
Hi, you're welcome to blog publicly about Copilot, and indeed many people have already done so. We partially address this point in our FAQ, [here](https://copilot.github.com/#faq-can-i-post-code-suggested-by-github-copilot-to-social-media-like-twitter) and [here](https://copilot.github.com/#faq-do-i-need-to-credit-github-copilot-for-helping-me-write-code).
